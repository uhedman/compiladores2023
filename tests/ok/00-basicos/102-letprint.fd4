let f (x:Nat) : Nat =
let x1 : Nat = print "x = " x in
let x2 : Nat = 2 in
let _ : Nat = print "x2 = " x2 in
let x3 : Nat = 3 in
let _ : Nat = print "x3 = " x3 in
let x4 : Nat = 4 in
let _ : Nat = print "x4 = " x4 in
let x5 : Nat = 5 in print "x1 = " x1

let y:Nat = f 1