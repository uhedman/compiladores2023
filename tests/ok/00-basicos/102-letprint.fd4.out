x = 1
x2 = 2
x3 = 3
x4 = 4
x1 = 1
